repo-projects/templating.proj

```diff
@@ -5,6 +5,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(DotNetBuildSourceOnly)' == 'true'">
+    <RepositoryReference Include="runtime" />
```

Comment:
Interesting that we hadn't seen the need for this yet.

Reply:
What requires this? These days runtime is only a toolset dependency in templating.